packages/contracts-bedrock/src/libraries/Preinstalls.sol

```diff
@@ -81,12 +87,18 @@ library Preinstalls {
 
+    bytes internal constant SenderCreator_v070Code =
```

Comment:
https://optimistic.etherscan.io/address/0xEFC2c1444eBCC4Db75e7613d20C6a62fF67A167C

Reply:
Confirmed that this matches